parcel/parcel.jl

```diff
@@ -0,0 +1,216 @@
+import OrdinaryDiffEq as ODE
+import CairoMakie as MK
+
+import Thermodynamics as TD
+import CloudMicrophysics as CM
+import CLIMAParameters as CP
+
+const CMT = CM.CommonTypes
+const CMO = CM.Common
+const CMI = CM.HetIceNucleation
+const CMP = CM.Parameters
+
+FT = Float64
+
+# Boiler plate code needed to have access to model parameters and constants
+include(joinpath(pkgdir(CM), "test", "create_parameters.jl"))
+toml_dict = CP.create_toml_dict(FT; dict_type = "alias")
+const param_set = cloud_microphysics_parameters(toml_dict)
+const thermo_params = CMP.thermodynamics_params(param_set)
+
+"""
+    ODE problem definition
+"""
+function cirrus_box(dY, Y, p, t)
+
```

Comment:
```suggestion
    (; param_set) = p
```
I'd recommend passing `param_set` through `p` to avoid the closure. We can also grab `FT`: `FT = eltype(Y)` (?) to avoid the closure over the global variable (which `FT` is a global non-constant, so this will _significantly_ slow things down).